frontend/src/lib/forms/UserTypeahead.svelte

```diff
@@ -12,29 +13,61 @@
   // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
   export let value: string;
   export let debounceMs = 200;
+  export let isAdmin: boolean = true;
```

Comment:
I'd like this to default to false if not specified, because we shouldn't just give people access to admin-only features **by default**. This will require going to AddOrgMemberModal and making sure that the `isAdmin` prop is set correctly where AddOrgMemberModal uses the UserTypeahead.

... Actually, I just realized that AddOrgMemberModal is going to need to be changed, which will include a change to the `_addOrgMember` function in the org page, and the SetOrgMemberRole method in OrgMutations.cs. There are quite a few changes needed there, so I'll go ahead and make them rather than asking you to do it.

Reply:
And it turns out that, no, AddOrgMemberModal is perfectly fine as is, because in AddOrgMemberModal, we're only providing site admins with the user typeahead — and site admins get to see users' email addresses, so the input field is being filled in with the email address just like it used to, and the GraphQL call works. So the only change that's actually needed in AddOrgMemberModal is that it should explicitly pass in a value for `isAdmin`, and the `isAdmin` prop here on line 16 should default to `false`.